climada/entity/impact_funcs/trop_cyclone.py

```diff
@@ -311,118 +436,66 @@ def get_countries_per_region(region=None):
-                "TJK", "TKM", "UGA", "ARE", "UZB", "YEM",
-            ],
-            "OC": [
-                "ASM", "AUS", "COK", "FJI", "PYF", "GUM", "KIR", "MHL", "FSM", "NRU",
-                "NCL", "NZL", "NIU", "NFK", "MNP", "PLW", "PNG", "PCN", "WSM", "SLB",
-                "TLS", "TKL", "TON", "TUV", "VUT", "WLF",
-            ],
-            "SI": [
-                "COM", "COD", "SWZ", "MDG", "MWI", "MLI", "MUS", "MOZ", "ZAF", "TZA",
-                "ZWE",
-            ],
-            "WP1": ["KHM", "IDN", "LAO", "MYS", "THA", "VNM"],
-            "WP2": ["PHL"],
-            "WP3": ["CHN"],
-            "WP4": ["HKG", "JPN", "KOR", "MAC", "TWN"],
-            "ROW": [
-                "ALB", "DZA", "AND", "AGO", "ATA", "AUT", "BLR", "BEL", "BEN", "BES",
-                "BIH", "BWA", "BVT", "BRA", "IOT", "BRN", "BGR", "BFA", "BDI", "CMR",
-                "CAF", "TCD", "CXR", "CCK", "COG", "HRV", "CUW", "CYP", "CZE", "CIV",
-                "DNK", "EGY", "GNQ", "EST", "FRO", "FIN", "FRA", "ATF", "GAB", "GMB",
-                "DEU", "GHA", "GIB", "GRC", "GRL", "GGY", "GIN", "GNB", "HMD", "VAT",
-                "HUN", "ISL", "IRL", "IMN", "ITA", "JEY", "KEN", "PRK", "XKX", "LVA",
-                "LSO", "LBR", "LBY", "LIE", "LTU", "LUX", "MLT", "MRT", "MYT", "MDA",
-                "MCO", "MNE", "MAR", "NAM", "NLD", "NER", "NGA", "MKD", "NOR", "PSE",
-                "POL", "PRT", "ROU", "RUS", "RWA", "REU", "BLM", "MAF", "SPM", "SMR",
-                "STP", "SEN", "SRB", "SYC", "SLE", "SGP", "SVK", "SVN", "SGS", "SSD",
-                "ESP", "SDN", "SJM", "SWE", "CHE", "TGO", "TUN", "TUR", "UKR", "GBR",
-                "UMI", "ESH", "ZMB", "ALA",
-            ],
-        }
-        # fmt: on
-        impf_id = {
-            "NA1": 1,
-            "NA2": 2,
-            "NI": 3,
-            "OC": 4,
-            "SI": 5,
-            "WP1": 6,
-            "WP2": 7,
-            "WP3": 8,
-            "WP4": 9,
-            "ROW": 10,
-        }
-        region_name = dict()
-        region_name["NA1"] = "Caribbean and Mexico"
-        region_name["NA2"] = "USA and Canada"
-        region_name["NI"] = "North Indian"
-        region_name["OC"] = "Oceania"
-        region_name["SI"] = "South Indian"
-        region_name["WP1"] = "South East Asia"
-        region_name["WP2"] = "Philippines"
-        region_name["WP3"] = "China Mainland"
-        region_name["WP4"] = "North West Pacific"
 
         if region == "all":
-            return region_name, impf_id, iso3n, iso3a
+            return (
+                country_code.region_name,
+                country_code.impf_id,
+                country_code.iso3n,
+                country_code.iso3a,
+            )
+
+        return (
+            country_code.region_name[region],
+            country_code.impf_id[region],
+            country_code.iso3n[region],
+            country_code.iso3a[region],
+        )
+
+    @staticmethod
+    def get_regions_per_countries(
```

Comment:
Can you please add tests for this method?

Reply:
yess :white_check_mark:

Reply:
Would the name `get_impf_id_regions_per_countries` be clearer? 

Just give it a thought: would you have found the use of this function with the current name if you had not written it just by looking at the class?